MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/handlers/setup/items/category/MaterialMultiLocationItem.java

```diff
@@ -169,25 +184,25 @@ public void onClick(InventoryClickEvent event) {
           switch(interactEvent.getAction()) {
             case PHYSICAL:
             case LEFT_CLICK_AIR:
-              new MessageBuilder("&c&l✘ &cYou can't use a location that is at your player location, please select the " + checkMaterial.toString() + "!").prefix().send(interactEvent.getPlayer());
+              new MessageBuilder("&c&l✘ &cYou can't use a location that is at your player location, please select the " + checkMaterials.toString() + "!").prefix().send(interactEvent.getPlayer());
               break;
             case LEFT_CLICK_BLOCK:
-              if(block.getType() != checkMaterial) {
-                new MessageBuilder("&c&l✘ &cPlease only use location where already is a " + checkMaterial.toString() + " to remove it as a " + name.toUpperCase() + "!").prefix().send(interactEvent.getPlayer());
+              if(!checkMaterials.contains(block.getType())) {
+                new MessageBuilder("&c&l✘ &cPlease only use location where already is a " + checkMaterials+ " to remove it as a " + name.toUpperCase() + "!").prefix().send(interactEvent.getPlayer());
```

Comment:
could return unreadable long message for user, getting the "clicked" type would be more accurate